week3/homework/src/index.js

```diff
@@ -1,3 +1,74 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+function readToDo(request, response) {
+  fs.readFile('./todos.json', 'utf8', (error, data) => {
+    if (error) {
+      console.log(error);
+      response.status(500).send('Problem Occurred.');
+    } else {
+      const index = request.params.id - 1;
```

Comment:
Check if the ID provided by use is within the range [1, todos-size], same note applies in `markAsDoneOrNotDone` function.